conf/api.go

```diff
@@ -394,20 +397,22 @@ func (confMap ConfMap) Dump() (buf string) {
 		for _, confOptionName = range confOptionNameSlice {
 			confOption = confSection[confOptionName]
 
-			buf += confOptionName + ":" + strings.Repeat(" ", confOptionNameLenMax-len(confOptionName))
+			buf.WriteString(confOptionName + ":" +
+				strings.Repeat(" ", confOptionNameLenMax-len(confOptionName)))
 
 			for confOptionValueIndex, confOptionValue = range confOption {
 				if 0 == confOptionValueIndex {
-					buf += " " + confOptionValue
+					buf.WriteString(" " + confOptionValue)
 				} else {
-					buf += ", " + confOptionValue
+					buf.WriteString(", " + confOptionValue)
 				}
 			}
 
-			buf += "\n"
+			buf.WriteString("\n")
 		}
 	}
 
+	confMapString = buf.String()
```

Comment:
Since you inexplicably renamed the return argument, what does this line accomplish?

Reply:
Now I see it... your use of buf as a strings.Builder nicely avoids changes throughout the code... then you "unwrap" it to return the desired string to callers with this line